p4include/v1model.p4

```diff
@@ -56,23 +56,51 @@ struct standard_metadata_t {
     //
     // flattening fields that exist in bmv2-ss
     // queueing metadata
-    @alias("queueing_metadata.enq_timestamp") bit<32> enq_timestamp;
-    @alias("queueing_metadata.enq_qdepth")    bit<19> enq_qdepth;
-    @alias("queueing_metadata.deq_timedelta") bit<32> deq_timedelta;
-    @alias("queueing_metadata.deq_qdepth")    bit<19> deq_qdepth;
+    @alias("queueing_metadata.enq_timestamp")
+    bit<32> enq_timestamp;
+    @alias("queueing_metadata.enq_qdepth")
+    bit<19> enq_qdepth;
+    @alias("queueing_metadata.deq_timedelta")
+    bit<32> deq_timedelta;
+    /// queue depth at the packet dequeue time.
+    @alias("queueing_metadata.deq_qdepth")
+    bit<19> deq_qdepth;
+
     // intrinsic metadata
-    @alias("intrinsic_metadata.ingress_global_timestamp") bit<48> ingress_global_timestamp;
-    @alias("intrinsic_metadata.egress_global_timestamp") bit<48> egress_global_timestamp;
-    @alias("intrinsic_metadata.lf_field_list") bit<32> lf_field_list;
-    @alias("intrinsic_metadata.mcast_grp")     bit<16> mcast_grp;
-    @alias("intrinsic_metadata.resubmit_flag") bit<32> resubmit_flag;
-    @alias("intrinsic_metadata.egress_rid")    bit<16> egress_rid;
+    @alias("intrinsic_metadata.ingress_global_timestamp")
+    bit<48> ingress_global_timestamp;
+    @alias("intrinsic_metadata.egress_global_timestamp")
+    bit<48> egress_global_timestamp;
+    /// Learn filter field list
+    @alias("intrinsic_metadata.lf_field_list")
+    bit<32> lf_field_list;
+    /// multicast group id (key for the mcast replication table)
+    @alias("intrinsic_metadata.mcast_grp")
+    bit<16> mcast_grp;
+    /// flag distinguishing original packets from resubmitted packets.
```

Comment:
The comment "flag distinguishing original packets from resubmitted packets" is not correct.  The field 'instance_type' is used for that purpose in v1model.

resubmit_flag is one of 4 fields that Antonin has suggested, and I like the idea, that they should be removed from standard_metadata in the near future, and instead made internal implementation details inside of BMv2.

One alternative is to remove the comment.  If you would prefer a more accurate one, I would replace it with something like this: "/// resubmit metadata field list id, or 0 if no resubmit operation has been performed"

Reply:
If you are at all curious to learn more about some of the BMv2 v1model implementation details, you can read them here: https://github.com/p4lang/behavioral-model/blob/master/docs/simple_switch.md

Reply:
There are some tests (that you wrote) which use resubmit_flag.
So for now I won't delete it; I will update the comment.
We can hopefully remove the field in a subsequent commit.

Reply:
That sounds fine to me.  We are not yet ready to delete the field -- it is still needed for BMv2 v1model to work correctly right now.  There are changes I have in process on a BMv2 PR that will make it unnecessary in the future, and we can remove it then.